app/src/main/assets/graphql/Example/Fragment/RepositoryFragment.graphql

```diff
@@ -0,0 +1,8 @@
+fragment RepositoryFragment on Repository {
```

Comment:
Example queries updated to make use of the externalized fragments. A more detailed note on this exists in the PR description.